<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wild7</class>
 <widget class="QWidget" name="Wild7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 7 Wild</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Profiles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>Ultra Sun</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="labelProfileTick">
        <property name="text">
         <string>Tick</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelProfileTickValue">
        <property name="text">
         <string>383e329</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelProfileShinyCharm">
        <property name="text">
         <string>Shiny Charm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="labelProfileShinyCharmValue">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="labelProfileOffset">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="labelProfileOffsetValue">
        <property name="text">
         <string>17</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxRNGInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>RNG Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_18">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStartDate">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="DateTimeEdit" name="dateTimeEditStartDate">
        <property name="displayFormat">
         <string>M/d/yyyy H:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEndDate">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="DateTimeEdit" name="dateTimeEditEndDate">
        <property name="displayFormat">
         <string>M/d/yyyy H:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelFrame">
        <property name="text">
         <string>Frame:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TextBox" name="textBoxStartFrame">
        <property name="text">
         <string notr="true">478</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="TextBox" name="textBoxEndFrame">
        <property name="text">
         <string notr="true">10000</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="pushButtonSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_20">
      <item row="0" column="0">
       <widget class="QLabel" name="labelEncounter">
        <property name="text">
         <string>Encounter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ComboBox" name="comboBoxEncounter">
        <item>
         <property name="text">
          <string>Grass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fish</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxSynch">
        <property name="text">
         <string>Synch</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxSynchNature"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelGenderRatio">
        <property name="text">
         <string>Gender Ratio</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="ComboBox" name="comboBoxGenderRatio"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBoxFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_19">
      <item row="0" column="1">
       <widget class="QLabel" name="labelNature">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Nature</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="CheckList" name="comboBoxNature"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonAnyNature">
        <property name="text">
         <string>Any</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelHiddenPower">
        <property name="text">
         <string>Hidden Power</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="CheckList" name="comboBoxHiddenPower"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonAnyHiddenPower">
        <property name="text">
         <string>Any</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelEncounterSlot">
        <property name="text">
         <string>Encounter Slots</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="CheckList" name="comboBoxEncounterSlot">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButtonAnyEncounterSlot">
        <property name="text">
         <string>Any</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelAbility">
        <property name="text">
         <string>Ability</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="ComboBox" name="comboBoxAbility">
        <item>
         <property name="text">
          <string>Any</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">H</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelGender">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="ComboBox" name="comboBoxGender">
        <item>
         <property name="text">
          <string>Any</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">♂</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">♀</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelShiny">
        <property name="text">
         <string>Shiny</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="checkBoxShiny">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="6">
       <widget class="IVFilter" name="ivFilter" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="TableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>IVFilter</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/IVFilter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckList</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/CheckList.hpp</header>
   <slots>
    <slot>resetChecks()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>DateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>Forms/Controls/DateTimeEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>dateTimeEditStartDate</tabstop>
  <tabstop>dateTimeEditEndDate</tabstop>
  <tabstop>textBoxStartFrame</tabstop>
  <tabstop>textBoxEndFrame</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>comboBoxEncounter</tabstop>
  <tabstop>checkBoxSynch</tabstop>
  <tabstop>comboBoxSynchNature</tabstop>
  <tabstop>comboBoxGenderRatio</tabstop>
  <tabstop>comboBoxNature</tabstop>
  <tabstop>pushButtonAnyNature</tabstop>
  <tabstop>comboBoxHiddenPower</tabstop>
  <tabstop>pushButtonAnyHiddenPower</tabstop>
  <tabstop>comboBoxEncounterSlot</tabstop>
  <tabstop>pushButtonAnyEncounterSlot</tabstop>
  <tabstop>comboBoxAbility</tabstop>
  <tabstop>comboBoxGender</tabstop>
  <tabstop>checkBoxShiny</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAnyNature</sender>
   <signal>clicked()</signal>
   <receiver>comboBoxNature</receiver>
   <slot>resetChecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1038</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>1012</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAnyHiddenPower</sender>
   <signal>clicked()</signal>
   <receiver>comboBoxHiddenPower</receiver>
   <slot>resetChecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1053</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>976</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAnyEncounterSlot</sender>
   <signal>clicked()</signal>
   <receiver>comboBoxEncounterSlot</receiver>
   <slot>resetChecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1068</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>934</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
